result/flow.yaml

```diff
@@ -173,18 +185,98 @@ components:
         latency:
           $ref: '#/components/schemas/Metric.Latency'
           x-field-uid: 14
-    Flow.MetricGroup:
+        tagged_metrics:
+          description: |-
+            List of tagged metrics
+          type: array
+          items:
+            $ref: '#/components/schemas/Flow.TaggedMetric'
+          x-field-uid: 15
+    Flow.TaggedMetric:
+      description: |-
+        A container for tagged flow metrics. The container is keyed by list of tags.
+      type: object
+      properties:
+        tags:
+          description: |-
+            List of tag and value pairs
+          type: array
+          items:
+            $ref: '#/components/schemas/Flow.MetricTag'
+          x-field-uid: 1
+        frames_tx:
+          description: >-
+            The current total number of frames transmitted
+          type: integer
+          format: uint64
+          x-field-uid: 2
+        frames_rx:
+          description: >-
+            The current total number of valid frames received
+          type: integer
+          format: uint64
+          x-field-uid: 3
+        bytes_tx:
+          description: >-
+            The current total number of bytes transmitted
+          type: integer
+          format: uint64
+          x-field-uid: 4
+        bytes_rx:
+          description: >-
+            The current total number of bytes received
+          type: integer
+          format: uint64
+          x-field-uid: 5
+        frames_tx_rate:
+          description: >-
+            The current rate of frames transmitted
+          type: number
+          x-field-uid: 6
+        frames_rx_rate:
+          description: >-
+            The current rate of valid frames received
+          type: number
+          x-field-uid: 7
+        loss:
+          description: >-
+            The percentage of lost frames
+          type: number
+          x-field-uid: 8
+        timestamps:
+          $ref: '#/components/schemas/Metric.Timestamp'
+          x-field-uid: 9
+        latency:
+          $ref: '#/components/schemas/Metric.Latency'
+          x-field-uid: 10
+    Flow.MetricTag:
       type: object
       properties:
         name:
           description: |-
-            Name of packet field metric group
+            Name of packet field metric tag
           type: string
           x-field-uid: 1
         value:
-          description: |-
-            Value of named packet field metric group
+          $ref: '#/components/schemas/Flow.MetricTag.Value'
+          x-field-uid: 2
+    Flow.MetricTag.Value:
+      description: |-
+        A container for metric tag value
+      type: object
+      properties:
+        choice:
+          description: Available formats for metric tag value
+          type: string
+          default: hex
```

Comment:
For query, need an option where value type won't be specified. An 'auto-type might work - which will be translated to a value matching with the tag.